### PR TITLE
Update funfacts.txt

### PR DESCRIPTION
**Pull request by UnderscoreRayless**

### CODE REVIEW

File: docs/assets/funfacts.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 RIP Technoblade\nHe will be missed
 RIP Technoblade\nHe will be missed
 RIP Technoblade\nHe will be missed
+RIP Light_Exists\nHe will be missed
```
Comment (KTibow) [inline docs/assets/funfacts.txt:4]: No results found. Empathy banana is here for you.

File: docs/assets/funfacts.txt [hunk 2]
```diff
@@ -10,22 +11,23 @@ At one point, Feather Client was willing to pay $300 USD to add Skytils.
 SkyblockPersonalized was once the cause of many rickrolls from right-clicking\nany of the Mathematics Farming Hoes. (It's been removed by now, of course.)
 The mod NEU has over 80k lines of code, making it\none of the largest mods out there for Skyblock.
 The icon you see in SkyClient's main menu and other places\nis made by none other than the esteemed Zetvue#9664.
-Most of SkyClient's staff have not touched Hypixel Skyblock for several months.
+Most of SkyClient's staff have not touched Hypixel Skyblock for several months. Quit today!
```
Comment (KTibow) [inline docs/assets/funfacts.txt:14]: does this go off screen

File: docs/assets/funfacts.txt [hunk 3]
```diff
@@ -10,22 +11,23 @@ At one point, Feather Client was willing to pay $300 USD to add Skytils.
...
-SBE has skidded from countless mods including DSM, Skytils and NEU!
 SkyClient contains only the finest ethically sourced mods and packs -\nwe asked permission before adding them to SkyClient.
 SkyClient is for more than Skyblock, even though it started out that way.
-SkyClient is a zero profit organization; we have 40 unpaid volunteers around the globe.
+SkyClient is a zero profit organization; we have unpaid volunteers around the globe.
 Most mods' settings are located in a command you can run when in game;\neg Patcher is configurable with `/patcher`.
 At one point in Minecraft, there was a rare chance of Snow Golems naturally spawning.
 When you're asked to update your mods, you can click on a mod's name to stop it\nfrom being replaced with SkyClient's latest version.
 W[h]yvest
+Quagsire
```
Comment (KTibow) [inline docs/assets/funfacts.txt:25]: who is this sire

File: docs/assets/funfacts.txt [hunk 4]
```diff
@@ -10,22 +11,23 @@ At one point, Feather Client was willing to pay $300 USD to add Skytils.
...
+Boost our discord!\n(https://inv.wtf/skyclient)
```
Comment (KTibow) [inline docs/assets/funfacts.txt:26]: ...

File: docs/assets/funfacts.txt [hunk 5]
```diff
@@ -10,22 +11,23 @@ At one point, Feather Client was willing to pay $300 USD to add Skytils.
...
+RUMOUR. Open your ears; 9r"5j5&?OWTY Z0d
```
Comment (KTibow) [inline docs/assets/funfacts.txt:27]: is this a reference to https://books.google.com/books?id=bZcKSdKK9NYC&pg=PA9&lpg=PA9&dq=open+your+ears+OWTY+Z0d&source=bl&ots=p9pzXPN4Dk&sig=ACfU3U2G05Kfgy45JEcxBZctRU-HE5OWcg&hl=en&sa=X&ved=2ahUKEwiOnsLOi6f6AhWKHjQIHRuTBpkQ6AF6BAgWEAM#v=onepage&q=open%20your%20ears%20OWTY%20Z0d&f=false

File: docs/assets/funfacts.txt (hunk 1 above)
Comment (KTibow) [reply docs/assets/funfacts.txt:4]: > <span>#</span>1 potato farmer
i knew him a little bit but not too well 
he comitted suicide to my knowledge 
i stumbled upon melonking's bio and decided to add it

File: docs/assets/funfacts.txt (hunk 2 above)
Comment (UnderscoreRayless) [reply docs/assets/funfacts.txt:14]: Try it and see!

Comment (UnderscoreRayless) [reply docs/assets/funfacts.txt:14]: If it does just remove it

Comment (UnderscoreRayless) [reply docs/assets/funfacts.txt:14]: It shouldn't as other lines are bigger then it (that aren't split with \n)

File: docs/assets/funfacts.txt (hunk 3 above)
Comment (UnderscoreRayless) [reply docs/assets/funfacts.txt:25]: bruh

File: docs/assets/funfacts.txt (hunk 4 above)
Comment (UnderscoreRayless) [reply docs/assets/funfacts.txt:26]: What's so wrong with boosting?!?!?!

File: docs/assets/funfacts.txt (hunk 5 above)
Comment (UnderscoreRayless) [reply docs/assets/funfacts.txt:27]: bro what

Comment (KTibow) [reply docs/assets/funfacts.txt:27]: what does this come from

File: docs/assets/funfacts.txt (hunk 4 above)
Comment (KTibow) [reply docs/assets/funfacts.txt:26]: advertisement

File: docs/assets/funfacts.txt (hunk 5 above)
Comment (UnderscoreRayless) [reply docs/assets/funfacts.txt:27]: you will never know

File: docs/assets/funfacts.txt (hunk 4 above)
Comment (UnderscoreRayless) [reply docs/assets/funfacts.txt:26]: you can't advertise in your own thing

Comment (KTibow) [reply docs/assets/funfacts.txt:26]: if you must then capitalize discord, give a reason or something, and remove parens

**Comment by nacrt** [comment]
terrible

**Comment by Ascynx** [comment]
Moment

**Comment by KTibow** [comment]
merge or no merge

**Pull request merged**